scripts/peering-prefix

```diff
@@ -140,6 +153,10 @@ withdraw    Withdraw prefix to one or all muxes.
             if using the appns submodule to avoid conflicting routing
             rules.
 
```

Comment:
Let's change the message below to "Change the announcements nexthop to force the mux to redirect packets into the user's container. Must be used with -m and with a mux running a container."